phpmyfaq/admin/assets/scss/layout/_configuration.scss

```
@@ -1,10 +1,12 @@
 //
 // Configuration
 //
+// Token-driven so the three themes stay consistent.
+//
```

Stylelint (17.11.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)